storm-core/src/clj/org/apache/storm/daemon/common.clj

```diff
@@ -408,19 +69,3 @@
                           (:default-shared-resources worker)
                           (:user-shared-resources worker)
                           ))
```

Comment:
If we are leaving some methods in clojure, could you file a follow on JIRA to remove the file once we have the other code ported over to java?